apps/admin/package.json

```
@@ -7,8 +7,10 @@
     "build": "next build",
     "clean": "rm -rf .next dist node_modules",
     "dev": "dotenv -e ../../.env next dev -- -p 3001",
-    "format": "prettier -w \"src/**/*.{ts,tsx,css,cjs,mjs,json}\"",
-    "lint": "eslint .",
+    "format": "prettier . --check --cache --cache-location='node_modules/.cache/prettiercache'",
```

IDK if this is just me, but this creates a new directory in a bunch of places that is named `'node_modules/.cache/prettiercache'`. I believe the intent here is to have the cache stored in the `node_modules/.cache/prettiercache`, which might make more sense to change this to 

```suggestion
"format": "prettier . --check --cache --cache-location=\"node_modules/.cache/prettiercache\"",
```

---

Are you on Windows? I wasn't aware of that, will change.